playbook/app/pb_kits/playbook/pb_multiple_users/multiple_users.rb

```diff
@@ -4,6 +4,9 @@ module Playbook
   module PbMultipleUsers
     class MultipleUsers < Playbook::KitBase
       prop :reverse, type: Playbook::Props::Boolean, default: false
+      prop :size, type: Playbook::Props::Enum,
+                  values: %w[xxs xs],
+                  default: "xs"
```

Comment:
Is there never a use case where we might want larger sizes?

Reply:
@thestephenmarshall Could conceivable end up adding larger sizes in the future but right now we only have kit styles that work with `xxs` and `xs` so it's limited to just those for now.

Reply:
Yeah i'd say we can revisit later, if that happens.